.github/automation/.drone.yml

```diff
@@ -113,3 +114,23 @@ steps:
   - .github/automation/build_acl.sh  --version 20.08 --arch arm64-v8a --root-dir $(pwd)/ComputeLibrary
   - .github/automation/build.sh --threading omp --mode Release --source-dir $(pwd) --build-dir $(pwd)/build --acl-dir $(pwd)/ComputeLibrary
   - .github/automation/test.sh --test-kind gtest --build-dir $(pwd)/build --report-dir $(pwd)/report
+
+---
+kind: pipeline
+name: Ubuntu18-JIT
+
+platform:
+  arch: amd64
```

Comment:
Is there Windows support for ARM build? I haven't found any Windows OS - specific pipelines for ARM architecture. The reason I am asking is that JIT generation should comply with OS - specific ABI and it would be good to test it.

Reply:
Thank you for the comment. I know there is Windows that works on Arm, but it is limited to AArch64 CPUs w/o SVE 512 instructions. I'd like to test SVE-enabled Windows in the future. Since ABI for Arm architecture is almost identical in Windows and Linux, I think our code may work on Windows as well.

Reply:
Thanks for the comment, I know there is a Windows that runs on Arm, but it is limited to the AArch64 CPU w/o SVE 512 instructions. I would like to test SVE-enabled Windows in the future. Since the Arm architecture's ABI is nearly identical on Windows and Linux, I think our code will work on Windows as well.

Reply:
Thanks for the information, sounds good to me!

Reply:
As far as I know there's currently no AArch64 Windows option available on Drone Cloud CI or Azure - so this is probably something to come back to.